server/runtime/TESTING.md

```diff
@@ -108,15 +108,49 @@ to pause a node and subsequently resume it.
 
 ## Testsuite Categories
 
-Tests are annotated with JUnit's `@Category` annotation
+Tests are annotated with JUnit's `@Category` annotation. The following categories are available:
+
+* `org.infinispan.server.test.category.Persistence`
+* `org.infinispan.server.test.category.Resilience`
+* `org.infinispan.server.test.category.Security`
+
+The default is to run all categories, but this can be overridden by setting the `defaultJUnitGroups` system property, e.g.
+
+`mvn -DdefaultExcludedJUnitGroups=org.infinispan.server.test.category.Persistence ...`
 
 ## Testsuite Properties
 
 The following is a list of properties which affect the build:
 
 * `org.infinispan.test.server.baseImageName` the base image to use for the server. Defaults to `jboss/base-jdk:11`.
 * `org.infinispan.test.server.driver`  the driver to use, `EMBEDDED` or `CONTAINER`. Defaults to the `EMBEDDED` driver.
+* `org.infinispan.test.server.extension.libs` locates artifact defined by G:A:V, you can pass a list of libraries (comma separeted) to be copied to the server. Only needed for container mode.
+* `org.infinispan.test.server.jdbc.database` database name to be used during persistence tests.
```

Comment:
NIT: the right parameter name is `org.infinispan.test.server.jdbc.databases`

Reply:
Fixed